src/app/pages/account-order-history/account-order-history-page.component.ts

```diff
@@ -17,12 +18,58 @@ export class AccountOrderHistoryPageComponent implements OnInit {
   orders$: Observable<Order[]>;
   ordersLoading$: Observable<boolean>;
   ordersError$: Observable<HttpError>;
+  moreOrdersAvailable$: Observable<boolean>;
+  filtersActive: boolean;
+  ordersAvailable = false;
+  private destroy$ = new Subject<void>();
 
   constructor(private accountFacade: AccountFacade) {}
 
   ngOnInit(): void {
-    this.orders$ = this.accountFacade.orders$({ limit: 30, include: ['commonShipToAddress'] });
+    this.orders$ = this.accountFacade.orders$;
     this.ordersLoading$ = this.accountFacade.ordersLoading$;
     this.ordersError$ = this.accountFacade.ordersError$;
+    this.moreOrdersAvailable$ = this.accountFacade.moreOrdersAvailable$;
+
+    this.loadOrders();
+
+    this.orders$?.pipe(takeUntil(this.destroy$)).subscribe(orders => {
+      this.ordersAvailable = orders.length > 0;
+    });
+  }
+
+  /**
+   * Load orders
+   */
+  loadOrders(): void {
```

Comment:
this method is not necessary and duplicates code: 
just call loadFilteredOrders({}) if you need the orders without a filter